app/assets/stylesheets/application.css

```diff
@@ -7,3 +7,19 @@
   width: 100px;
   height: 100px;
 }
+
```

Comment:
- [ ] Nice job so far on introducing improvements 👏 However, let's take note that it's one of the project's objectives that our design closely mimics the given [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=). Let's therefore ensure that the _typography_, _layout_ and _color_ on the splash screen is similar to the original design.

**Your design:**
**Your splash screen:**     |  **Expected**     
:-------------------------:|:-------------------------:
<img width="800" alt="Home Page" src="https://github.com/M-Anwar-Hussaini/Expense-Management/assets/12745474/7300a07f-1e28-4223-be4d-a2df62f98a2b"> | <img width="800" alt="Home Page" src="https://github.com/M-Anwar-Hussaini/Expense-Management/assets/12745474/b7ed973c-85be-4bbd-8a52-5b33bbc686b1">

Reply:
Hi @rloterh . 
Thanks, I'm going to handle it right now.